index.ts

```diff
@@ -386,8 +348,8 @@ class Analytics {
         Authorization: 'Basic ' + Buffer.from(`${this.writeKey}:`).toString('base64'),
       },
       body: JSON.stringify(data),
-      retryDelay: this._exponentialDelay.bind(this),
-      retryOn: this._isErrorRetryable.bind(this),
+      retryDelay: this.getExponentialDelay.bind(this),
+      retryOn: this.isErrorRetryable.bind(this),
     };
 
     if (this.timeout) {
```

Comment:
this check no longer does anything, since `timeout` is officially never null

Reply:
Hm, that's a good point. Perhaps we want null to mean "no timeout, do whatever Node/the OS does".